forge-gui/res/cardsfolder/upcoming/trouble_in_pairs.txt

```diff
@@ -0,0 +1,10 @@
+Name:Trouble in Pairs
+ManaCost:2 W W
+Types:Enchantment
+R:Event$ BeginTurn | ActiveZones$ Battlefield | ValidPlayer$ Opponent | ExtraTurn$ True | Skip$ True | Description$ If an opponent would begin an extra turn, that player skips that turn instead.
+T:Mode$ AttackersDeclared | AttackingPlayer$ Opponent | CheckSVar$ X | SVarCompare$ GE2 | Execute$ TrigDraw | TriggerZones$ Battlefield | TriggerDescription$ Whenever an opponent attacks you with two or more creatures, draws their second card each turn, or casts their second spell each turn, you draw a card.
+SVar:X:Count$ValidAll Creature.attackingYou
```

Comment:
where are you getting such weird ValidAll ideas? :P
use IsPresent and don't forget NoResolvingCheck please...

Reply:
Actually you can just use ValidAttackers + ValidAttackersAmount for a cleaner result

Reply:
Unless I scripted it wrong it gave an out of bounds error when I was attack by one creature. `Range [2, 1) out of bounds for length 1`

Reply:
ugh I'll check

Reply:
well `ValidAttackers$ Creature.attackingYou | ValidAttackersAmount$ GE2` works fine for me